src/interfaces/IEthMultiVault.sol

```diff
@@ -748,9 +872,114 @@ interface IEthMultiVault {
     /// @return assets number of assets user has in the vault
     function getVaultStateForUser(uint256 vaultId, address receiver) external view returns (uint256, uint256);
 
+    /// @notice returns the shares for recipient and other important values when depositing 'assets' into a bonding curve vault
```

Comment:
3.10 - New *Curve Functions in EthMultiVault Missing From Interface

We have added the new curve related methods to the interface here.